Library/brew.rb

```diff
@@ -94,14 +94,8 @@ def require?(path)
     exit Homebrew.failed? ? 1 : 0
   else
     require "tap"
-    possible_tap = case cmd
-    when "brewdle", "brewdler", "bundle", "bundler"
-      Tap.fetch("Homebrew", "bundle")
-    when "cask"
-      Tap.fetch("caskroom", "cask")
-    when "services"
-      Tap.fetch("Homebrew", "services")
-    end
+    possible_tap = OFFICIAL_CMD_TAPS.find { |_, repo| repo == cmd }
```

Comment:
I also agree with @xu-cheng that it would be better to use his suggested hash approach from the start. Unlike the currently proposed solution it doesn't make any assumptions about the tap name vs. command name relationship and allows for multiple commands in a single tap, thus feels very future-proof to me. It doesn't feel like “doing it right” from the start creates a significant burden.